src/pages/profile/ProfileView.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, useState } from 'react';
```

Comment:
po co ten komponent skoro na naszym widoku edycji jednoczesnie wyswietlamy aktualne dane i dajemy mozliwosc ich edytowania? Imo to jest zbedne

Reply:
jw ;)